### ISSUE TITLE
Sculpt is in the newest version 1.0.1 invisible

### ISSUE THREAD

**Issue by Titoo8899** [Lemonszz/Statues#6]
Please fill out this template when submitting an issue.

---
**Mod Version**:  
1.12.2:  1.0.1
**Forge Version**:
1.12.2:  14.23.5.2771

**This issue occurs on the lastest version of Forge? (Y/N)**:
N
**Crash Log (Post to Pastebin/Other Text hosting)**: 
Give it not the mod have not a crash problem
**What happens**:
Then I play the 1.0.0 version it work normal but the newest 1.0.1 version have one bug. The Sculpt is invisible then we will edit and then press sculpt. In all times we see this invisible status. And the colour see we not in the 1.0.1 version please fix this. I have 2 Pictures to see the problem all 2 Pictures was with the newest 1.0.1 version of your mod make!!!
![2019-01-26_12 41 05](https://user-images.githubusercontent.com/42325138/51786711-7ed95b00-2168-11e9-9102-7302d071b941.png)
![2019-01-26_12 41 16](https://user-images.githubusercontent.com/42325138/51786712-80a31e80-2168-11e9-9135-543399f3fb3c.png)

**Comment by Titoo8899** [comment]
Screenshot 1 is the edit more of all parts of the statue but you can not see. The Statue is invisible. 
Screenshot 2 is then we have finish the edit and sculpt click. The person is invisible. We can only the hitbox see of this. Please fix this invisible problem.

**Comment by Titoo8899** [comment]
My guess this is one configs bug in your script.

**Comment by ZombieHDGaming** [comment]
Yeah, I'm gonna say this is a mod conflict on your end, cause.....
![2019-01-26_09 52 27](https://user-images.githubusercontent.com/10361287/51788834-8f7dd700-2150-11e9-9ccc-f22c4162f34e.png)
![2019-01-26_09 52 41](https://user-images.githubusercontent.com/10361287/51788839-97d61200-2150-11e9-935b-5b68160c0e44.png)
No issue in my modpack on the latest mod release. Another mod in your pack is causing an issue, and this does not seem to be a direct Statues issue.

**Comment by Titoo8899** [comment]
I give you a link to my Google Drive Mods folder and you can check what mod this problem make.
Today was 5 new Mods.

New Mods 26.01.2019:
- Smooth Font (Latest mod what I have download)
- Xaeros Minimap
- DynamicStealth
- statues-rechiseled (your mod 1.0.1 version)
- CustomDailyReward

Link: []([]([](https://drive.google.com/file/d/1Py-tfuTL4iQCrvCDk-tYclW0fk32VI_P/view?usp=sharing[](url))))

**Comment by ZombieHDGaming** [comment]
Try removing DynamicStealth. It seems like it is doing some weird things.

**Comment by Titoo8899** [comment]
I have written the person what he must fix her mod of Dynamic Stealth and hope the next version of Dynamic Stealth work with this mod normal

**Comment by ZombieHDGaming** [comment]
Was that the issue then?

**Comment by Titoo8899** [comment]
I have your mod delete why I with the 0.35 version of Dynamic Stealth mod the same problems have.

**Comment by ZombieHDGaming** [comment]
What? I can't understand what you're trying to say

**Comment by Laike-Endaril** [comment]
[https://github.com/Laike-Endaril/Dynamic-Stealth/issues/13](https://github.com/Laike-Endaril/Dynamic-Stealth/issues/13)

I'll look into this on my end.  And yeah, DS does a *lot* of "weird things" in order to work correctly.  I'm actually surprised at how few mod conflicts I've gotten reports of so far.

I'll make sure to post here if I find new information and/or resolve this from my end.

**Comment by Laike-Endaril** [comment]
Confirmed this happens with all combinations of (DS 035, DS 036) and (Statues 1.0.0, Statues 1.0.1).  No errors recorded.

I'll start looking through source, but I'm guessing this is happening because statues uses an EntityLivingBase to render the statue, and said EntityLivingBase is encapsulated inside blocks that don't let light in, ie. it is in light level 0, which would make it invisible in accordance with DS's sight algorithm.

**Comment by Titoo8899** [comment]
You say with DS 34 works normal and the other 2 newest versionen DS35 +36 work not.

**Comment by Laike-Endaril** [comment]
This should now be fixed in DS version 037
